src/components/pacbio/PacbioRunInfoEdit.vue

```diff
@@ -85,13 +85,21 @@ export default {
   },
   computed: {
     ...mapGetters(['runItem', 'smrtLinkVersionList', 'smrtLinkVersion', 'instrumentType']),
+    // Makes an array of objects with value and text properties to make
+    // the options of smrt-link-version select drop-down list.
     smrtLinkVersionSelectOptions() {
-      // Returns an array of objects with value and text properties to make
-      // the options of smrt-link-version select drop-down list.
-      return Object.values(this.smrtLinkVersionList).map(({ id, name }) => ({
-        value: id,
-        text: name,
-      }))
+      var activeVersionsList = []
```

Comment:
would it be possible just to use.
    ```
  return Object.values(this.smrtLinkVersionList).map(({ id, name }) => ({
        value: id,
        text: name,
      }).filter( version => version.active)

```
 I am not sure just a thought?

Reply:
Yes probably, I thought there would be a better way! Thanks